source/java-drivers.txt

```diff
@@ -6,6 +6,17 @@ MongoDB Java Drivers
 
 .. default-domain:: mongodb
 
+.. facet::
+  :type: programming_language
+  :values: java
+
+.. facet::
+  :type: genre
+  :values: reference
+
+.. meta:: 
+  :keywords: java sync, java async
```

Comment:
S: The async driver is also called "reactive streams"--probably good to add here

Reply:
Keywords are flexible, so I'll add this one.  Do you think that reactive streams is better than java async, or do users use both terms?